flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/hybrid/enumerator/ArcticEnumeratorOffset.java

```diff
@@ -63,6 +63,12 @@ public void snapshotTimestampMs(Long snapshotTimestamp) {
     this.snapshotTimestampMs = snapshotTimestamp;
   }
 
+  public boolean isEmpty() {
```

Comment:
It seems there are two definitions of empty `ArcticEnumeratorOffset`:`ArcticEnumeratorOffset.empty()` and `ArcticEnumeratorOffset.EMPTY`.
Should we keep just one and make the `isEmpty` method depend on that?